src/main/java/com/getindata/connectors/http/HttpSinkPostRequestCallback.java

```diff
@@ -0,0 +1,32 @@
+package com.getindata.connectors.http;
+
+import java.io.Serializable;
+import java.net.http.HttpResponse;
+import java.util.Map;
+
+import com.getindata.connectors.http.internal.sink.HttpSinkRequestEntry;
+import com.getindata.connectors.http.internal.table.sink.HttpDynamicSink;
+
+/**
+ * An interface for a custom class that logs responses and requests processed by the HTTP Sink.
+ *
+ * <p>{@link HttpDynamicSink} processes responses that it gets from the HTTP endpoint along their
+ * respective requests. One can customize the behaviour of such a processor by implementing both
+ * {@link HttpSinkPostRequestCallback} and {@link HttpSinkPostRequestCallbackFactory}.
+ */
+public interface HttpSinkPostRequestCallback extends Serializable {
```

Comment:
I would like to suggest removing "Sink" from name of this interface to have `HttpSinkPostRequestCallback`
We may use it in Lookup source someday.

Reply:
done